primitives/staking/src/offence.rs

```diff
@@ -91,14 +91,21 @@ pub trait Offence<Offender> {
 	) -> Perbill;
 }
 
+/// Errors that may happen on offence reports.
+#[derive(PartialEq, sp_runtime::RuntimeDebug)]
+pub enum OffenceError {
+	/// The report has already been sumbmitted.
+	DuplicateReport,
```

Comment:
Do we at least want to have a variant that may takes a `u8` or whatever? Otherwise we will need to update this for every error that may come u.

Reply:
I think that `Other(u8)` should be added when the need arises, r.n. it would be just an unused variant?

Reply:
I will remind you on your next pr :P

Reply:
Ok, added it here